examples/javascript/button.js

```diff
@@ -26,7 +26,7 @@
 var groveSensor = require('jsupm_grove');
```

Comment:
@jontrulson you're right, @SuyashLakhotia any chance you can open this as an issue instead please?